galaxy/main/models/collection.py

```diff
@@ -21,6 +21,7 @@
 from django.core.validators import MinValueValidator, MaxValueValidator
 from django.db import models
 from pulpcore.app import models as pulp_models
+import semantic_version as semver
```

Comment:
Whats the reason for importing `semantic_version` as `semver`?

Since we previously used `semver` but stopped, seeing code that uses `semver.Version` initially reads as a bug.
If there is no reason not to, my preference would be to just `import semantic_version` and use `semantic_version.Version()` below

Reply:
We have it like this in a few other places, maybe left overs from updating only the imported package, I will change it here.

Reply:
`semver` is not only a module of the same name package, but a commonly used abbreviation to "Semantic version". We dropped `semver` package entirely and it's not present anywhere in our dependencies, so I think it's still acceptable to use `semver` just as an abbreviation to the "Semantic version" term.

Reply:
Import 'semantic_version as semver' makes it unnecessarily more difficult to find actual use of the `semantic_version` module  (ie, 'git grep semantic_version' only finds the import).

Reply:
It's ok to use aliases in python. We use lots of aliases for example for `rest_framework`. You wouldn't find `rest_framework` usages by just grep-ing the source code.